reserve-watch/bot-config.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Same comment about formatting this configuration file to more closely resemble our current convention.

If the effort produces no better readability, perhaps it can remain as is.